multi-node-k8s.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/sh
+
+# TODO: disable debugging
+set -x
+
+# get latest stable version, strip the leading "v"
+K8S_VERSION="$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)"
+
+# configure cluster parameters
+WORKERS=2
+WORKER_MEM=5
+NO_OF_DISKS=2
+DISK_SIZE=25
+MASTER_MEM=5
+MASTER_CPU=4
+
+# enable additional sources for yum
+# (epel for ansible and golang)
+yum -y install epel-release
+
+# Install additional packages
+yum -y install \
+	qemu-kvm \
+	qemu-kvm-tools \
+	qemu-img \
+	libvirt \
+	ansible \
+	golang
+
+# install kubectl
+curl -LO https://storage.googleapis.com/kubernetes-release/release/"${K8S_VERSION}"/bin/linux/amd64/kubectl
+mv kubectl /usr/bin/kubectl
+chmod +x /usr/bin/kubectl
+
+# Vagrant needs libvirtd running
+systemctl start libvirtd
+
+# Log the virsh capabilites so that we know the
+# environment in case something goes wrong.
+virsh capabilities
+
+# TODO: this is not the right way to install vagrant on CentOS, but SCL only
+# provides vagrant 1.x and we need >= 2.2
+yum -y install https://releases.hashicorp.com/vagrant/2.2.7/vagrant_2.2.7_x86_64.rpm
+go get github.com/ceph/go-ceph
```

Comment:
you do not need to get go-ceph, it is part of the `vendor/` directory of ceph-csi